resources/settings.xml

```diff
@@ -5,9 +5,10 @@
     <setting id="chromeos_version" value="" visible="false"/>
     <category label="30900"> <!-- Troubleshooting -->
         <setting label="30901" type="action" id="info" action="RunScript(script.module.inputstreamhelper, info)"/>
-        <setting label="30903" help="30904" type="bool" id="disabled" value="false"/>
+        <setting label="30903" help="30904" type="bool" id="disabled" value="false" default="false"/>
         <setting label="30904" type="text" enable="false"/> <!-- disabled_warning -->
         <setting label="30905" help="30906" type="action" id="install_widevine" action="RunScript(script.module.inputstreamhelper, widevine_install)" enable="eq(-2,false)" visible="!system.platform.android"/>
         <setting label="30907" help="30908" type="action" id="remove_widevine" action="RunScript(script.module.inputstreamhelper, widevine_remove)" enable="eq(-3,false)" visible="!system.platform.android"/>
+        <setting label="30909" type="folder" id="temp_dir" source="" option="writeable" default="special://profile/addon_data/script.module.inputstreamhelper"/>
```

Comment:
This option should really only be visible on ARM. There is no need to modify the path on other platforms IMO.

Reply:
Why not call this `temp_path` instead? We use that internally as well.